app/assets/stylesheets/pages/_index.scss

```diff
@@ -1,2 +1,3 @@
 // Import page-specific CSS files here.
 @import "home";
+@import "new-van";
```

Comment:
It's better to have a file '_vans.scss' where we will have all css related to vans pages

Reply:
i don't understand what i have to change

Reply:
Instead of _new-van.scss, rename it in _vans.scss so that we can add all css related to van pages